### PR TITLE
(CFACT-214) Use Myget mirror for Chocolatey pkgs

### PR DESCRIPTION
**Pull request by MikaelSmith**
Use Myget mirror for Chocolatey packages with installers added to the
Nuget packages to avoid dependencies on other external servers (such as
Sourceforge).

### CODE REVIEW

**Comment by MikaelSmith** [comment]
ping @melissa

**Comment by melissa** [comment]
cool, looks good

File: contrib/cfacter.ps1 [hunk 1]
```diff
@@ -47,17 +47,16 @@ $yamlPkg = "${yamlCppVer}-${mingwVer}"
 ### Setup, build, and install
 ## Install Chocolatey, then use it to install required tools.
 iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
-choco install 7zip.commandline -Version 9.20.0.20130618
-choco install cmake -Version 3.0.2
-choco install git.install -Version 1.9.5
-choco install python -Version 3.4.2
-choco install doxygen.install -Version 1.8.8
```
Comment (melissa) [inline contrib/cfacter.ps1]: so we no longer need python and doxygen?

Comment (MikaelSmith) [inline contrib/cfacter.ps1]: Python is used for cpplint, and doxygen for producing documentation. I guess we probably want those on build machines, so I'll add them back.

Comment (melissa) [inline contrib/cfacter.ps1]: I'm not worried about doxygen, but python might be good to have still

**Comment by melissa** [comment]
Actually, while you're in here, would you be willing to also run the tests?

**Comment by MikaelSmith** [comment]
I've run these these with arch=32 and arch=64. Are those the tests you mean?

**Comment by MikaelSmith** [comment]
Oh, you mean add ctest?

**Comment by melissa** [comment]
Yeah, add ctest :)

**Comment by MikaelSmith** [comment]
Moved ctest to a different PR for CFACT-202.

File: appveyor.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 install:
   - git submodule update --init --recursive
 
-  - ps: choco install 7zip.commandline
-  - ps: choco install ruby -Version 2.1.5
-  - ps: choco install mingw -Version 4.8.3.20141208
+  - ps: choco install 7zip.commandline -source https://www.myget.org/F/puppetlabs/api/v2
+  - ps: choco install ruby -Version 2.1.5 -source https://www.myget.org/F/puppetlabs/api/v2
+  - ps: choco install mingw-w64 -Version 4.8.3 -source https://www.myget.org/F/puppetlabs/api/v2
```
Comment (ferventcoder) [inline appveyor.yml]: Did you find api/v2/ necessary for installs?

Comment (MikaelSmith) [inline appveyor.yml]: Oh, didn't test. I'll try it out.

**Pull request merged**